fileServer.js

```diff
@@ -0,0 +1,48 @@
+ 'use strict';
+
+ var http = require('http');
+ var fs = require('fs');
+ var path = require('path');
+ var url = require('url');
+ var rootDir = __dirname + '/public';
+
+
+ const mimeType = {
+ 	'html': 'text/html',
+ 	'jpeg': 'image/jpeg',
+ 	'jpg': 'image/jpeg',
+ 	'png': 'image/png',
+ 	'js': 'text/javascript',
+ 	'css': 'text/css'
+ };
+
+ function send404Request(res) {
+ 	fs.readFile('./404.html', 'UTF-8', function(err, html) {
```

Comment:
This does not appear to use your rootDir - all static files should be under `public`